core/templates/dev/head/pages/exploration_editor/translation_tab/state_translation_directive.html

```diff
@@ -147,10 +150,18 @@
 </div>
 
 <style>
+  .oppia-translation-page-statename {
+    margin-bottom: 80px;
+    margin-left: 80px;
+    margin-right: auto;
+    margin-top: 40px;
+    position: absolute;
+  }
   .oppia-state-translation-page {
     display: flex;
     flex-direction: column;
     min-height: 500px;
+    margin-top: 75px;
```

Comment:
Move this to line 163. (ma >  mi)

Reply:
Hi @DubeySandeep  can you please tell me how to  make it align of the state component navbar?

Reply:
I'm solve it.